website/docs/guides/sl-snowflake-qs.md

```diff
@@ -818,7 +818,7 @@ After setting up your orders model:
 
 ```yaml
 semantic_models:
-  - name: customers
+  - name: dim_customers
```

Comment:
@mirnawong1 can't add a comment on the right line for some reason! But in line 826 of this file the model: key should say model: ref('dim_customers') and it currently says ref('customers') I think line 821 can stay as customers if we like, it's just that it has to ref dim_customers.